libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,38 @@
+/** Anime DTO. */
+export type AnimeDto = {
+	readonly id: number;
+	readonly created: string;
+	readonly modified: string;
+	readonly title_eng: string;
+	readonly title_jpn: string;
+	readonly image: string;
+	readonly aired: AiredDto;
+	readonly type: AnimeTypeDto;
+	readonly status: AnimeStatusDto;
+	readonly score: number;
+	readonly user_score: number;
+	readonly studios: readonly number[];
+	readonly genres: readonly number[];
+};
```

Comment:
Consider this, would be more readable
```suggestion
export type AnimeDto = Readonly<{
	id: number;
	created: string;
	modified: string;
	title_eng: string;
	title_jpn: string;
	image: string;
	aired: AiredDto;
	type: AnimeTypeDto;
	status: AnimeStatusDto;
	score: number;
	user_score: number;
	studios: readonly number[];
  genres: readonly number[];
}>;